.azure-pipelines/docker-sonic-mgmt-mixed.yml

```diff
@@ -57,10 +57,10 @@ stages:
         git submodule update --init --recursive -- src/sonic-platform-daemons src/sonic-genl-packet src/sonic-sairedis src/ptf src/sonic-device-data src/sonic-dash-api
 
         make SONIC_BUILD_JOBS=$(nproc) DEFAULT_CONTAINER_REGISTRY=publicmirror.azurecr.io ENABLE_DOCKER_BASE_PULL=y configure PLATFORM=generic
-        make -f Makefile.work BLDENV=bullseye SONIC_BUILD_JOBS=$(nproc) DEFAULT_CONTAINER_REGISTRY=publicmirror.azurecr.io ENABLE_DOCKER_BASE_PULL=y LEGACY_SONIC_MGMT_DOCKER=n target/docker-sonic-mgmt.gz
+        make -f Makefile.work BLDENV=bullseye SONIC_BUILD_JOBS=$(nproc) DEFAULT_CONTAINER_REGISTRY=publicmirror.azurecr.io ENABLE_DOCKER_BASE_PULL=y LEGACY_SONIC_MGMT_DOCKER=y target/docker-sonic-mgmt.gz
         cp target -r $(Build.ArtifactStagingDirectory)/target
         docker load -i target/docker-sonic-mgmt.gz
-        docker tag docker-sonic-mgmt $REGISTRY_SERVER/docker-sonic-mgmt:py3only
+        docker tag docker-sonic-mgmt $REGISTRY_SERVER/docker-sonic-mgmt:mixed
```

Comment:
Just wondering if the `mixed` tag is relevant only to release branches? If so, should this pipeline's branch include specify something like `2024??` or `202*` ?

```
trigger: none
pr:
  branches:
    include:
    - 2024??
  paths:
    include:
    - dockers/docker-sonic-mgmt
```

Reply:
Yes, it's prepared for 202205 and earlier branches, but we can still use master branch to run the docker build pipeline, and we will stop building new mixed images once build succeeded